### ISSUE TITLE
Rendering error on `Tabs` when `initialPage` is not `0`

### ISSUE THREAD

**Issue by boringer** [GeekyAnts/NativeBase#1318]
## react-native, react and native-base version

React Native: 0.48.4.

React: 16.0.0-alpha.12.

Native Base: 2.3.3.

## Steps to reproduce (code snippet or screenshot)

```js
import React from 'react';
import { StyleSheet, Text, View } from 'react-native';
import {
    Container,
    Tabs,
    Tab,
} from 'native-base';

export default class App extends React.Component {
    render() {
        return (
            <Container>
                <Tabs initialPage={1}>
                    <Tab heading="Tab 1">
                        <Text>1</Text>
                    </Tab>
                    <Tab heading="Tab 2">
                        <Text>2</Text>
                    </Tab>
                    <Tab heading="Tab 3">
                        <Text>3</Text>
                    </Tab>
                </Tabs>
            </Container>
        );
    }
};
```

Result:

![img_ca5acb1f7d9c-1](https://user-images.githubusercontent.com/14685751/31849579-0e31479e-b677-11e7-9ada-5073c690767b.jpeg)

However, when I add a padding to the parent (`<Container style={{paddingHorizontal: 10}}>`), it works fine:

![img_66f986ad364d-1](https://user-images.githubusercontent.com/14685751/31849606-721ee7fc-b677-11e7-926c-3debbd8f6e32.jpeg)

## Is the bug present in both ios and android or in any one of them?

Both.

**Comment by dtuite** [comment]
I'm seeing the same thing with

```
    "react-native": "^0.48.4",
    "native-base": "^2.3.2",
    "react": "^16.0.0",
```

**Comment by ziudeso** [comment]
Temporary workaround but it works (RN: 0.48):
    "expo": "^21.0.0",
    "moment": "^2.18.1",
    "native-base": "^2.3.2",
    "react-native": "https://github.com/expo/react-native/archive/sdk-21.0.2.tar.gz"

XXX = src/dist
in node_modules/native-base/XXX/basic/tabs/index.js
Replace:

```
if (this.scrollView && Platform.OS === "android") {
	const x = this.props.initialPage * this.state.containerWidth;
	this.scrollView.scrollTo({ x, animated: false });
}
```
with:
```
if (this.scrollView) {
	const x = this.props.initialPage * this.state.containerWidth;
	this.scrollView.scrollTo({ x, animated: false });
}
```

**Comment by toupilsner** [comment]
@ziudeso solution helps with selecting correct tab, but my problems is that the view is not drawn, and I have to touch and move the screen before data in Tab is visible. Tabs are nested in a TabNavigator, and they are the second tab in TabNavigator

`"@expo/samples": "^2.0.2",
    "expo": "^22.0.0",
    "moment": "^2.19.1",
    "native-base": "^2.3.2",
    "react": "16.0.0-beta.5",
    "react-native": "https://github.com/expo/react-native/archive/sdk-22.0.1.tar.gz",
    "react-navigation": "^1.0.0-beta.15",
    "react-redux": "^5.0.6",
    "redux": "^3.7.2",
`


**Is the bug present in both ios and android or in any one of them?**
Only IOS

**Comment by SupriyaKalghatgi** [comment]
#1010

**Comment by SupriyaKalghatgi** [comment]
@vijaya06 Check for Tabs issues, if you could any help from it
Else go ahead and raise issue

**Comment by hanssonfredrik** [comment]
This issue is not fixed in native-base version 2.3.10.

**Comment by yaronlevi** [comment]
Same here for native-base@2.2.2 and RN@0.49.0
